packages/components/src/input-control/reducer/reducer.ts

```diff
@@ -192,10 +185,12 @@ export function useInputControlStateReducer(
 		dispatch( {
 			type,
 			payload: { value: nextValue, event },
-		} );
+		} as actions.InputAction );
```

Comment:
Same here I guess. What's the error if we don't explicitly cast this object?

Reply:
It complains that the `type` parameter must be `INVALIDATE` for some reason. I'm not sure why it complains here and not in the other places 🤔